pkg/cache/v3/user.go

```diff
@@ -82,7 +81,7 @@ func (u *userAttributesCache) GetUserAttributeKeyAll(environmentId string) ([]st
 	return attributeKeys, nil
 }
 
-func (u *userAttributesCache) Put(userAttributes *userproto.UserAttributes) error {
+func (u *userAttributesCache) Put(userAttributes *userproto.UserAttributes, ttl int) error {
```

Comment:
This way, the caller can use 5 * time.Minute, 30*time.Second, etc., without having to remember “this int is in seconds.

```suggestion
func (u *userAttributesCache) Put(userAttributes *userproto.UserAttributes, ttl time.Duration) error {
```

Reply:
Thank you for the good suggestion!
I changed the ttl argument to time.Duration type.
https://github.com/bucketeer-io/bucketeer/pull/2029/commits/9fca7379a7456cef44d9597c224639a446caa300